### ISSUE TITLE
Decommission the old Slack workspace

### ISSUE THREAD

**Issue by JeanChristopheMorinPerso** [AcademySoftwareFoundation/rez#1457]
We used to have a dedicated Slack workspace before our migration to ASWF. We should probably decommission it since it's entirely inactive and all our activities now happen in ASWF slack workspace.

**Comment by instinct-vfx** [comment]
We had a total of 15k messages. Over the course of its existance there was a steady growth in activity that lead up to the very active channel we know have in ASWF. Thank you to everyone who contributed over the years and see you soon in the new workspace :)

I have created and downloaded a backup of the history. I don't want to publish that publicly but i wonder if there is any place we want to keep this that is not public?

**Comment by instinct-vfx** [comment]
Workspace was deleted

**Comment by instinct-vfx** [comment]
Thanks for closing. I moved it to done, but that only updates the board, not the ticket. Memo to self written.

**Comment by JeanChristopheMorinPerso** [comment]
Yeah, the automation is reversed. SO if you close an issue, it's be moved to done in the project. But the other way around doesn't work by default.
